src/posthog-core.js

```diff
@@ -354,10 +357,9 @@ PostHogLib.prototype._send_request = function (url, data, options, callback) {
     var DEFAULT_OPTIONS = {
         method: this.get_config('api_method'),
         transport: this.get_config('api_transport'),
+        verbose: this.get_config('verbose') || data['verbose'],
```

Comment:
This is the only change that wasn't clear to me, can you provide some context?

Reply:
So xhr has some weird callback logic (which I'm hoping to someday kill). If verbose_mode is on, here the callback argument changed:

https://github.com/PostHog/posthog-js/pull/144/files#diff-b2cef6371e5f8c8df5bc261c250273b3407a01e4503b087d64ac0f63d710c69cL487-L491

Previously `verbose_mode` was populated like this (on the left):
https://github.com/PostHog/posthog-js/pull/144/files#diff-b2cef6371e5f8c8df5bc261c250273b3407a01e4503b087d64ac0f63d710c69cL383-L386

Now it gets populated here and passed to `xhr` via options argument.

There should be no behavioral change, just some more clarity.